code/modules/mob/living/living.dm

```diff
@@ -1752,7 +1752,8 @@
 	. = buckled
 	buckled = new_buckled
 	if(buckled)
-		ADD_TRAIT(src, TRAIT_IMMOBILIZED, BUCKLED_TRAIT)
+		if(!istype(buckled,/obj/vehicle/ridden/wheelchair))
```

Comment:
Dumb way of doing this, hard-coded type paths are funky and this gives no explanation as to *why* this is here.

Make this a trait on wheelchairs instead that this checks for.